### PR TITLE
Reduced repetition in drivetrain code

### PR DESCRIPTION
**Pull request by siIverfish**
I made a method that returns all of the encoders in the drivetrain, and changed `resetEncoders` and `setAllEncoders` to use this method and a foreach loop instead of the same code 4 times.

### CODE REVIEW

**Comment by ebay-kid** [comment]
`getAllEncoders` does not compile, since array literals are defined with curly braces

File: src/main/java/frc/robot/subsystems/DriveTrainSubsystem.java [hunk 1]
```diff
@@ -137,24 +146,15 @@ public double findZRotationSpeedFromAngle(double angle) {
...
-      frontLeftEncoder, 
-      frontRightEncoder, 
-      rearLeftEncoder, 
-      rearRightEncoder
-    );
-  }
-
   public void resetEncoders() {
-    for (RelativeEncoder encoder : getAllEncoders())
+    for (RelativeEncoder encoder : encoders)
```
Comment (ebay-kid) [inline src/main/java/frc/robot/subsystems/DriveTrainSubsystem.java:150]: Use `this.encoders`

File: src/main/java/frc/robot/subsystems/DriveTrainSubsystem.java [hunk 2]
```diff
@@ -137,24 +146,15 @@ public double findZRotationSpeedFromAngle(double angle) {
...
     {
       encoder.setPosition(0);
     }
   }
 
   public void setAllEncoders(double position) {
-    for (RelativeEncoder encoder : getAllEncoders())
+    for (RelativeEncoder encoder : encoders)
```
Comment (ebay-kid) [inline src/main/java/frc/robot/subsystems/DriveTrainSubsystem.java:157]: Same here, `this.encoders`

**Review by ebay-kid — COMMENTED** [review]
Just these

**Comment by siIverfish** [comment]
Ok I fixed my bad code so it shouldn't break anymore.

**Pull request merged**